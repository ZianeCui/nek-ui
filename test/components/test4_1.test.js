import { mount } from 'vue-test-utils'
import Test4 from '../../components/Example/test4.vue'

describe('使用mount测试组件', () => {
    let cmp

    beforeEach(() => {
        cmp = mount(Test4, {
            // `propsData` ??
            propsData: {
                list: ['Cat']
            }
        })
    })

    it('has received ["Cat"] as the message property', () => {
        expect(cmp.vm.list).toEqual(['Cat'])
    })

    it('has the expected html structure', () => {
        expect(cmp.element).toMatchSnapshot()
    })
})