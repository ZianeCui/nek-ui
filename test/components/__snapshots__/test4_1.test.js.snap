// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`使用mount测试组件 has the expected html structure 1`] = `
<ul>
  <ul>
    <li
      class="test3"
    >
      
        Cat
    
    </li>
  </ul>
</ul>
`;
